CRMImporter/FieldMap.cs

```diff
@@ -10,15 +10,15 @@ public class FieldMap
     {
         public string SourceField;
         public string TargetField;
-        public IConverter Convert;
+        public IConverter[] Convert;
 
         /// <summary>
         /// FieldMap constructor
         /// </summary>
         /// <param name="source">The field in your source data to match against</param>
         /// <param name="target">The field in CRM to match to</param>
         /// <param name="converter">Optional conversion function</param>
-        public FieldMap(string source, string target, IConverter converter = null)
+        public FieldMap(string source, string target, params IConverter[] converter)

```

Comment:
The way that params work is that if it isn't specified then it will be an empty array. https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params